airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1402,6 +1402,30 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/state/create_or_update:
```

Comment:
@evantahler @davinchia discussion from parent PR:
![image](https://user-images.githubusercontent.com/7798112/186284356-daea1b17-7490-44d5-9060-1830d3748459.png)

Reply:
Resolving as our plan is to leave this as-is, but feel free to reopen if we want to discuss further!